x/cli/version_command.go

```diff
@@ -0,0 +1,25 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+)
+
+var Version = "dirty"
+
+type versionCommand struct {
+	name    string
+	version string
+}
+
+func (vc *versionCommand) WriteSynopsis(io.Writer) (int, error) { return 0, nil }
+
+func (vc *versionCommand) WriteHelp(w io.Writer) (int, error) {
+	return io.WriteString(w, "Print the app version")
```

Comment:
```suggestion
	return io.WriteString(w, "Prints the app version")
```

Reply:
Do we have to put an `s` ? It's the imperative form, isn't it ?

Reply:
I would say third person, "this command prints the app version",  but that might be my broken angliche talking :)

Reply:
I think the imperative form would fit better as those are commands you want to execute (like an order). It is also the standard form in all the helpers I know.

Reply:
Okay nvn